Documentation/etcd-mixin/mixin.libsonnet

```diff
@@ -219,6 +219,32 @@
               message: 'etcd instance {{ $labels.instance }} HTTP requests to {{ $labels.method }} are slow.',
             },
           },
+         {
+            alert: 'etcdDatabaseSizeAlmostFull',
+            expr: |||
+              (etcd_mvcc_db_total_size_in_bytes/etcd_server_quota_backend_bytes)*100 > 95
+            ||| % $._config,
+            'for': '10m',
+            labels: {
+              severity: 'critical',
+            },
+            annotations: {
+              message: 'etcd cluster "{{ $labels.job }}": database size is almost full on etcd instance {{ $labels.instance }}, please defrag or increase the quota as the writes to etcd will be disabled when it is full.',
```

Comment:
nit: `database size almost exceeds the defined quota on ...`

Reply:
Fixed it, thanks.